README.md

```diff
@@ -11,11 +11,17 @@ Two sample projects (Sbt and Maven) which demonstrate recommended configuration
 This sample demonstrates how to conditionally depend on all modules. If use only on some of the modules just edit the `libraryDependencies` definition accordingly. If you are just looking for a copy&paste snippet for your `build.sbt` file, here it is:
 
 ```scala
-// add scala-xml dependency when needed (for Scala 2.11 and newer) in a robust way
+// add scala-xml dependency when needed (for Scala 2.12 and newer) in a robust way
 // this mechanism supports cross-version publishing
 // taken from: http://github.com/scala/scala-module-dependency-sample
 libraryDependencies := {
   CrossVersion.partialVersion(scalaVersion.value) match {
+    // if scala 2.12+ is used, use scala-swing 2.x
+    case Some((2, scalaMajor)) if scalaMajor >= 12 =>
+      libraryDependencies.value ++ Seq(
+        "org.scala-lang.modules" %% "scala-xml" % "1.0.3",
+        "org.scala-lang.modules" %% "scala-parser-combinators" % "1.0.3",
```

Comment:
I was trying to isolate the changes to 2.12, but I can bump these up while we're at it.  I'll just need to retest.